Documentation/fleet_vs_kubernetes.md

```diff
@@ -0,0 +1,28 @@
+# fleet vs kubernetes Comparison Table
+
+Here is a short comparison table which will help you to choose which tool is more suitable for you:
```

Comment:
This table briefly compares `fleet` to Kubernetes to help inform deployment decisions: